recipes/sensor.rb

```diff
@@ -58,6 +58,17 @@
   runit_service 'mss' do
     action :enable
   end
+
+  creds = data_bag_item('credentials', 'metaflows')
+
+  template '/etc/rc0.d/K01delete-metaflows-sensor' do
```

Comment:
K is executed on runlevel exit and S on runlevel enter right? Do you mean for this to run on entering the halt state?